design-system/Text/Text.presets.ts

```diff
@@ -0,0 +1,26 @@
+import { TextStyle } from "react-native";
+
+import { $baseStyle, $fontWeightStyles, $sizeStyles } from "./Text.styles";
+import { ThemedStyleArray } from "../../theme/useAppTheme";
+
+export type IPresets =
+  | "default"
+  | "bold"
+  | "heading"
+  | "subheading"
+  | "formLabel"
+  | "formHelper";
+
+export const $presets: Record<IPresets, ThemedStyleArray<TextStyle>> = {
```

Comment:
What's the $ prefix for? Not used to seeing that outside the scope of observables / jQuery

Reply:
Removed!

It's how InfiniteRed do styling with Ignite.

So that it's easier for auto complete for example you type $ and see all the styling available.

I removed it for now until we all agree on this way of working.

I don't want this to class with other things in the app